### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by tanyapowell** [tanyapowell/restaurant-reviews#2]
This task is dependent on #1. Once it has been merged into the `main` branch this can be picked up

### Description
A `README` should be added to the repository to provide information of what the project is, how to run the project, how to contribute and who the main contributors are.

### Acceptance Criteria
`README` must include the following at minimum
- What the project is
- The technology used in the project
- How to run the project
- How to contribute
- The main contributors
